cmd/minishift/cmd/root_test.go

```diff
@@ -83,6 +86,10 @@ func TestPreRunDirectories(t *testing.T) {
 	// Need to create since Minishift config get created in root command
 	os.Mkdir(filepath.Join(testDir, "config"), 0755)
 	os.Mkdir(filepath.Join(testDir, "machines"), 0755)
+
+	allInstanceConfigPath := filepath.Join(testDir, "config", "allinstances.json")
```

Comment:
on a side note, eventually I'd like to move _allinstances.json_ out of ~/.minishift. Maybe have a ~/.minishiftrc or something. Not an issue atm.

Reply:
@hferentschik can you motivate this a little? Please create an issue so this can be tracked outside of this PR.